Source/Core/DolphinWX/Config/SlippiPlaybackConfigPane.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <wx/arrstr.h>
+#include <wx/panel.h>
+#include <wx/textctrl.h>
+
+enum TEXIDevices : int;
+
+class wxButton;
+class wxCheckBox;
+class wxChoice;
+class wxDirPickerCtrl;
+class wxSpinCtrl;
+class wxString;
+class wxStaticText;
+class wxTextCtrl;
+
+class SlippiPlaybackConfigPane final : public wxPanel
```

Comment:
Lets move this whole class into SlippiConfigPane and rename the other class in that one to SlippiNetplayConfigPane